packages/kyt-core/src/config/webpack.dev.server.js

```diff
@@ -11,7 +11,7 @@ const {
 const StartServerPlugin = require('./StartServerPlugin');
 const getPolyfill = require('./getPolyfill');
 
-const nodeArgs = ['-r', 'source-map-support/register'];
+const nodeArgs = ['-r', 'source-map-support/register', '--max_old_space_size=4096'];
```

Comment:
Not sure if it really likes it this way or not, I'm guessing not but jic, here it is
```suggestion
const nodeArgs = ['-r', 'source-map-support/register', '--max_old_space_size', '4096'];
```